cmd/osv-scanner/internal/testcmd/run.go

```diff
@@ -20,3 +23,36 @@ func Run(t *testing.T, tc Case) (string, string) {
 
 	return stdout.String(), stderr.String()
 }
+
+func Run(t *testing.T, tc Case) {
+	t.Helper()
+
+	stdout, stderr := run(t, tc)
+
+	if tc.isOutputtingJSON() {
```

Comment:
do we want to factor in if there are any JSON replacement rules? I think doing so technically would be more performant, as `normalizeJSON` always calls `json.Indent` even if there are no rules, though that's probably not even measurable....

Reply:
I think that's fine to always call json.Indent, it'll make the diffs easier to read as well (maybe? I don't quite remember how the diffs work for JSON).